### ISSUE TITLE
7-Row keyboard trackpoint non-functional

### ISSUE THREAD

**Issue by Mikejmoffitt** [hamishcoleman/thinkpad-ec#6]
Following the recommended procedure, where pins 25, 27, and 29 are covered on the X220 keyboard's internal connector, the trackpoint buttons do not work with the X220 keyboard on an X230.

**Comment by Mikejmoffitt** [comment]
In addition, attempting to revert to the original keyboard is difficult, as I can not seem to re-flash the embedded controller with the stock firmware.

**Comment by hamishcoleman** [comment]
I assume you got it all working?  was there anything we could add to the instructions that would have helped you to succeed faster?

**Comment by Mikejmoffitt** [comment]
I fixed the trackpoint buttons, but closed the issue as that's likely more hardware related, given the success others have had. Regarding reverting, it would be nice to have a make target to create a bootable image that just writes the original EC firmware, but I managed to do so myself with a little hack so the X230 is at least usable.

I figured further discussion and troubleshooting would be most appropriate in the thinkpads.com thread, since I doubt this is really a software issue.
